trie/triedb/pathdb/disklayer.go

```diff
@@ -331,14 +331,20 @@ func (dl *diskLayer) commit(bottom *diffLayer, force bool) (*diskLayer, error) {
 	if overflow {
 		if _, ok := dl.buffer.(*nodebufferlist); ok {
 			persistentID := rawdb.ReadPersistentStateID(dl.db.diskdb)
-			if persistentID > limit {
-				oldest = persistentID - limit + 1
-				log.Info("Forcing prune ancient under nodebufferlist", "disk_persistent_state_id",
-					persistentID, "truncate_tail", oldest)
-			} else {
-				log.Info("No prune ancient under nodebufferlist, less than db config state history limit")
+			if limit >= persistentID {
+				log.Info("No prune ancient under nodebufferlist, less than db config state history limit", "persistent_id", persistentID, "limit", limit)
```

Comment:
Maybe can lower `INFO` level log to `DEBUG` due to high frequency.

Reply:
updated